common/install_utils.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -ex
+
+# Install common dependencies
+spack add cmake
```

Comment:
Does this install the latest version of cmake for x-distro?

Reply:
It installs the latest stable release if the version if not specified